docs/receiving/verifying-payloads/how-manual.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Verifying Manually
+---
+
+import CodeTabs from '@theme/CodeTabs';
+import TabItem from '@theme/TabItem';
+
+The recommended way to verify webhooks is using our official libraries as outlined in the [How to Verify](./how.mdx) section.
+
+However, here are instructions for verifying manually for those who need it.
+
+## Verifying signatures manually
+
+Each webhook call includes three headers with additional information that are used for verification:
+
+* `Svix-Id`: the unique message identifier for the webhook message. This identifier is unique across all messages, but will be the same when the same webhook is being resent (e.g. due to a previous failure).
+* `Svix-Timestamp`: timestamp in [seconds since epoch](https://en.wikipedia.org/wiki/Unix_time).
+* `Svix-Signature`: the [Base64](https://en.wikipedia.org/wiki/Base64) encoded signature.
+
+### Constructing the signed content
+
+The signature is composed by concatenating the following the id, timestamp and payload, separated by the full-stop character `.`. Which in code will look something like:
```

Comment:
```suggestion
The signature is composed by concatenating the id, timestamp and payload, separated by the full-stop character (`.`). In code, it will look something like:
```